var/spack/repos/builtin/packages/py-basemap/package.py

```diff
@@ -22,7 +24,11 @@ class PyBasemap(PythonPackage):
     depends_on('py-setuptools', type=('build', 'run'))
     depends_on('py-numpy', type=('build', 'run'))
     depends_on('py-matplotlib', type=('build', 'run'))
-    depends_on('py-pyproj@:1.99', type=('build', 'run'))
+    depends_on('py-pyproj@:1.99', type=('build', 'run'), when='@:1.2.0')
+    # 1.2.1 is PROJ6 compatible
+    # https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=939022
+    depends_on('py-pyproj@2:', type=('build', 'run'), when='@1.2.1:')
```

Comment:
According to the `requirements.txt`, the supported versions of these packages are:
```
matplotlib>=1.0.0,!=3.0.1
numpy>=1.2.1
pyproj>=1.9.3
pyshp>=1.2.0
six
```
Can you update these constraints to reflect that? It looks like Python 2.6+ is also required, although I don't know how many people care about pre-2.6.